aten/src/ATen/core/jit_type.h

```diff
@@ -1386,6 +1386,12 @@ struct getTypePtr_<int64_t> final {
   }
 };
 template <>
+struct getTypePtr_<c10::ScalarType> final {
+  static TypePtr call() {
+    return IntType::get();
```

Comment:
wait, what is it? isn't scalar like a union of int and float?

Reply:
Yeah you're right, I think this should be `ScalarTypeType::get()`

Reply:
Looks like I have to fix up other existing erroneous sites also, e.g. https://github.com/pytorch/pytorch/blob/master/torch/csrc/jit/frontend/schema_type_parser.cpp#L39

Reply:
You're thinking of `Scalar` which is `NumberType::get()`

Reply:
also, this is `ScalarType`, not `Scalar`, lol this is confusing